controllers/common/hcoConditions.go

```diff
@@ -59,3 +59,11 @@ func (hc HcoConditions) GetCondition(conditionType string) (metav1.Condition, bo
 	cond, found := hc[conditionType]
 	return cond, found
 }
+
+func (hc HcoConditions) IsStatusConditionTrue(conditionType string) bool {
+	if hc.HasCondition(conditionType) {
+		return hc[conditionType].Status == metav1.ConditionTrue
+	}
+
+	return false
```

Comment:
With this implementation, the code access the map twice to get the same information. You can access the map only once. e.g.
```suggestion
    cond, found := hc[conditionType]
    return found && cond.Status == metav1.ConditionTrue
```

Reply:
Done